Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -322,7 +322,7 @@ extension PreplannedMapArea: PreplannedMapAreaProtocol {
     }
 }
 
-private extension FileManager {
+package extension FileManager {
```

Comment:
The `pakage` modifier is used for exposing a declaration across targets of the same package without making it public. That doesn't apply in this situation, so it doesn't make sense to use it. Remember, use the lowest access level possible for a declaration. In this case, the extension should be internal (no access level keyword). Then, since this extension is no longer private to this file, move it into a new file in this same directory called something like `FileManager+Offline.swift`. And now that they are no longer private, you can write unit tests for the enclosing declarations. Yay!